apps/omar-upload-app/grails-app/conf/application.yml

```diff
@@ -1,141 +1,49 @@
 ---
-logging:
-  config: classpath:logback.groovy
-
-# Set the context path for the application.
-#server:
-#  contextPath: /omar-upload
-
-# Define the http protocols that swagger can use
 swagger:
```

Comment:
Is this config file also pushed to the dev config repo?

Reply:
The needed stuff is. The application.yml is defining the defaults with parameters so they should work anywhere